### PR TITLE
Modify UG formatting

### LINKED ISSUE #153: [PE-D] Date is not required in input
This application is supposed to be a platform that helps drivers and passengers of any IT company quickly arrange carpooling in order to lower their carbon footprint. Hence, it would seem important for a date to be used (instead of Days) because else they cannot specify which week they want to carpool. 

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Ellevy/ped#6

### LINKED ISSUE #176: [PE-D] Inconsistent use of grey highlight
This image mentioned that grey highlights were used for commands.

But as you can see in from image, it was used to highlight a file name instead.

I think there's more instances of such inconsistency throughout the UG.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: deyixtan/ped#1

### LINKED ISSUE #191: [PE-D] Suggestion for unpool header name
I think this should be `Remove pool` instead of `Remove pools` since it `unpool` doesn't support MassOps.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: deyixtan/ped#16

### LINKED ISSUE #212: [PE-D] Target user
Not sure how the features in this app cater to IT companies more than other companies.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: lirc572/ped#18

### PR DESCRIPTION
**Pull request by ZechariahTan**
* Address issue #153
* Address issue #212
* Close issue #191

### CODE REVIEW

**Review by lyueyang — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ GreenMileageEfforts (GME) is a platform that helps drivers and passengers of any
 # 1. Introduction
 
 ## 1.1 About Green Mileage Efforts
-Green Mileage Efforts(GME) is a holistic carpooling management solution designed to help corporations reduce their carbon footprint. The GME system allows for the simple creation and management of groups of employees looking to carpool to and from their office. Through the GME system, users can find employees based on their carpooling preferences and quickly group them with drivers. The system also maintains a database of the arranged carpooling groups for easy management.
+Green Mileage Efforts(GME) is an effecient carpooling management solution designed to help corporations reduce their carbon footprint. The GME system allows for the simple creation and management of weekly carpooling groups of employees looking to carpool to and from their office. These [pools](#6-glossary) of employees can then carpool from the office regularly on the specified days and times every week. Through the GME system, users can find employees based on their carpooling preferences and quickly group them with drivers. The system also maintains a database of the arranged carpooling groups for easy management.
```
Comment (lyueyang) [inline docs/UserGuide.md]: Spelling mistake for efficient here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -300,8 +307,9 @@ Action | Format, Examples
 
 Term used | Meaning
 --------|------------------
-Pool | A group of employees carpooling together. Consists of one driver and at least one passenger.
+Pool | A group of employees carpooling together. Consists of one driver and at least one passenger. The pools generated for a specifc day and time are the same every week unless reorganised by the user. 
 Passenger | An employee carpooling with at least one driver.
-TripDay | Day of intended carpooling trip 
+TripDay | Day of the intended carpooling trip.  
 TripTime | Time of intended carpooling trip
```
Comment (lyueyang) [inline docs/UserGuide.md]: Full stop for TripDay but none for TripTime, and also TripTime now doesnt have an extra 'the'

**Review by lyueyang — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -116,11 +116,18 @@ Adds a new passenger in the GME terminal.
 
 **Format:** `add n/NAME p/PHONE a/ADDRESS d/TRIPDAY t/TRIPTIME [tag/TAG tag/TAG ...]`
 
+<div markdown="block" class="alert-warning">
+
+**:warning: As the current iteration of GME(v1.4) is still only a minimum viable product, every passenger can only specify one day and one time a week for when they intend to carpool.**
```
Comment (chewterence) [inline docs/UserGuide.md]: should this be note instead of warning?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ Please note the following symbols used in the User Guide which may serve as poin
 
 ### 3.1.1 Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explianing how to access the help page.
```
Comment (lyueyang) [inline docs/UserGuide.md]: 'explaining' is mispelled here

**Review by lyueyang — CHANGES_REQUESTED** [review]

**Pull request merged**